db/migrate/20200622155632_repository_table_state_update.rb

```diff
@@ -8,18 +8,35 @@ class TempStateModel < ApplicationRecord
   def up
     TempStateModel.find_each do |table_state|
       state = table_state.state
-      order_state = state['order'][0][0]
-      state['order'][0][0] = order_state + 2 if order_state > 5
+      order_state = state.dig('order', 0, 0)
+
+      unless order_state
+        order = state.dig('order', '0')
```

Comment:
isn't it already checked in order_state?

Reply:
There is old state format, where some fields use Hash instead of Array